python/tvm/contrib/download.py

```diff
@@ -23,16 +23,18 @@ def download(url, path, overwrite=False, size_compare=False):
 
     size_compare : bool, optional
         Whether to do size compare to check downloaded file.
-    """
 
-    import requests
+    verbose: int, optional
```

Comment:
use logging

Reply:
This is used in other scripts,  which does not set `logging.basicConfig`. And this function will draw a 
 progress bar, which cannot be done by logging.  I prefer to not using logging.